packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -430,19 +434,19 @@ export class GroupedListSection extends React.Component<IGroupedListSectionProps
    */
   private _updateDroppingState = (newIsDropping: boolean, event: DragEvent): void => {
     const { isDropping } = this.state;
-    const { dragDropEvents } = this.props;
+    const { dragDropEvents, group } = this.props;
 
-    if (!isDropping) {
-      if (dragDropEvents && dragDropEvents.onDragLeave) {
-        dragDropEvents.onDragLeave!(event, undefined);
-      }
-    } else {
-      if (dragDropEvents && dragDropEvents.onDragEnter) {
-        dragDropEvents.onDragEnter(event, undefined);
+    if (isDropping !== newIsDropping) {
+      if (isDropping) {
```

Comment:
I assume this logic has always been incorrectly inverted based on your changes?

Reply:
Yes, this might cause issue to someone who has taken a dependency on the old logic - but I feel that might not be the case. For example, dragEnter earlier was called very late (when dragLeave should be called). I am not sure if people would be taking dependency on that

Reply:
Your fix seems correct, so let's just note it in case we need to revert / hotfix.